docs/feature_bsp_del.md

```diff
@@ -0,0 +1,23 @@
+# Backspace Delete
+
+## Usage
+
+`KC_BSDL` will function as normal `KC_BSPC` when pressed. When either `SHIFT` key is held, it will function as `KC_DEL` instead. When both `KC_RSHIFT` and `KC_LSHIFT` are held, the output will be `SHIFT` modifier with `KC_DEL` pressed.
+
+## What Your OS Sees
+
+If Mary presses BSDL on her keyboard, the OS will see an `KC_BSPC` character. Now if Mary holds Shift down and presses BSDL it will output `KC_DEL`. If she holds down both Shift keys, it will output `SHIFT` and `KC_DEL`.
```

Comment:
```suggestion
If Mary presses `BSP_DEL` on her keyboard, the OS will see an `KC_BSPC` character. Now if Mary holds Shift down and presses `BSP_DEL` it will output `KC_DEL`. If she holds down both Shift keys, it will output Shift and `KC_DEL`.
```